### ISSUE TITLE
TypeError: 'int' object is not iterable

### ISSUE THREAD

**Issue by 0xdevalias** [dcoker/awsmfa#4]
Python 3.x
```
⇒  awsmfa
The config profile (identity) could not be found
Available profiles: default, josh
Traceback (most recent call last):
  File "/Users/devalias/.pyenv/versions/3.5.2/bin/awsmfa", line 9, in <module>
    load_entry_point('awsmfa==0.2.7', 'console_scripts', 'awsmfa')()
  File "/Users/devalias/.pyenv/versions/3.5.2/lib/python3.5/site-packages/awsmfa/__main__.py", line 42, in main
    status = one_mfa(args, credentials)
  File "/Users/devalias/.pyenv/versions/3.5.2/lib/python3.5/site-packages/awsmfa/__main__.py", line 73, in one_mfa
    session, session3 = make_session(args.identity_profile)
TypeError: 'int' object is not iterable
```

Python 2.x
```
⇒  awsmfa
The config profile (identity) could not be found
Available profiles: default, josh
Traceback (most recent call last):
  File "/Users/devalias/.pyenv/versions/2.7.12/bin/awsmfa", line 9, in <module>
    load_entry_point('awsmfa==0.2.7', 'console_scripts', 'awsmfa')()
  File "/Users/devalias/.pyenv/versions/2.7.12/lib/python2.7/site-packages/awsmfa/__main__.py", line 42, in main
    status = one_mfa(args, credentials)
  File "/Users/devalias/.pyenv/versions/2.7.12/lib/python2.7/site-packages/awsmfa/__main__.py", line 73, in one_mfa
    session, session3 = make_session(args.identity_profile)
TypeError: 'int' object is not iterable
```

**Comment by dcoker** [comment]
d78456f9 should replace the unhelpful stack trace with something friendlier. Released as 0.2.8. Thanks for the bug report!

**Comment by 0xdevalias** [comment]
Ah, that makes sense now! Thanks :)
